tests/features/tmgmt/tmgmt_poetry.feature

```diff
@@ -17,6 +17,26 @@ Feature: TMGMT Poetry features
       | de        |
       | it        |
 
+  Scenario Outline: Checking automatic counter request.
+    Given I am logged in as a user with the 'administrator' role
+    And I go to "node/add/page"
+    And I fill in "Title" with "<title>"
+    And I fill in "Body" with "<body>"
+    And I press "Save"
+    And I select "Published" from "state"
+    And I press "Apply"
+    Then I click "Translate" in the "primary_tabs" region
+    And I select the radio button "" with the id "edit-languages-fr"
+    And I press "Request translation"
+    And I press "Submit to translator"
+    And I store the job reference of the translation request page
+    Then the poetry translation service received the translation request
```

Comment:
And you are completely right. The only thing is that this functionality is covered in poetry client implementation and to cover it with test we should create a ticket. There is more things that we can check on poetry client part thus I would like to propose follow up ticket to cover more of them. At that time tmgmt_poetry client was not covered at all thus it is a technical debt that we need to cover.